SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier3/GalvanicCore.cs

```diff
@@ -114,33 +106,16 @@ private void RecalculateStatsPreventHealing(ILContext il)
 
         private void FinalBuffStackLostPreventHealing(On.RoR2.CharacterBody.orig_OnBuffFinalStackLost orig, CharacterBody self, BuffDef buffDef)
         {
-            //float? health = null;
-
             if (buffDef == SS2Content.Buffs.bdGalvanized)
             {
-                //health = self.healthComponent.health;
-                //self.SetAmputateMaxHealth(self.amputatedMaxHealth - healthPenaltyCalculated);
-                //SS2Log.Warning("awwawa : " + self.healthComponent.health + " | " + self.cursePenalty);
                 var token = self.gameObject.GetComponent<GalvanizedMarker>();
```

Comment:
Change this logic into using "EnsureComponent"
https://github.com/TeamMoonstorm/MoonstormSharedUtils/blob/82ec6a0202e3bed4dfc0b88ef6caf9b54efb2a42/Runtime/Code/MSUtil.cs#L118